src/routes/webhooks/paddle.ts

```diff
@@ -1030,6 +1032,12 @@ export const processTransactionUpdated = async ({
 
 export const paddle = async (fastify: FastifyInstance): Promise<void> => {
   fastify.register(async (fastify: FastifyInstance): Promise<void> => {
+    fastify.addHook('onRequest', async (request, res) => {
+      if (isProd && !remoteConfig.vars.paddleIps?.includes(request.ip)) {
```

Comment:
if its empty lets skip check, so we have an option to pass everything through if needed